sympy/stats/tests/test_continuous_rv.py

```diff
@@ -318,6 +318,13 @@ def test_gamma_inverse():
     X = GammaInverse("x", a, b)
     assert density(X)(x) == x**(-a - 1)*b**a*exp(-b/x)/gamma(a)
 
+def test_gompertz():
+    b = Symbol("b", positive=True)
+    eta = Symbol("eta", positive=True)
+
+    X = Gompertz("x", b, eta)
+    assert density(X)(x) == b*eta*exp(b)*exp(-b*exp(eta*x))*exp(eta*x)
```

Comment:
This is what the ```pdf``` functions says: 

```python
    def pdf(self, x):
        eta, b = self.eta, self.b
        return b*eta*exp(b*x)*exp(eta)*exp(-eta*exp(b*x))
```

This is what IPython gives me when I execute density(X)(x):

```python
In [15]: density(X)(x)
Out[15]: b*eta*exp(b)*exp(-b*exp(eta*x))*exp(eta*x)
```